servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -104,7 +106,36 @@ If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
 |`version`
 |Any valid Docker tag
 |Docker tag to use with Istio images.
-|0.1.0
+|0.2.0
+|===
+
+.Kiali parameters
+|===
+|Parameter |Values |Description  |Default
+
+|`prefix`
+|Any valid image repo.
+|Which prefix to apply to the Kiali image name used in docker pull.
+|If deployment_type=origin the default is `kiali/`.
```

Comment:
s/deployment_type=origin/\`deployment_type=origin\` (two places)

Reply:
This should now be done